app/Support/Shortcode/Shortcode.php

```diff
@@ -196,6 +196,10 @@ private function renderLink(ShortcodeInterface $shortcode): string
         // Determine the URL from the shortcode's 'url' parameter or content.
         $url = $shortcode->getParameter('url') ?: $shortcode->getContent();
 
+        if (empty($url)) {
+            return '';
```

Comment:
Would it be better to return something indicating a problem occurred? i.e. "[unsupported link]" or "[broken link]"?

Reply:
No objections here. Updated in latest to return "[broken link]".